kernel/univ.mli

```diff
@@ -319,15 +319,24 @@ module AUContext :
 sig 
   type t
 
+  val repr : t -> UContext.t
+  (** [repr ctx] is [(Var(0), ... Var(n-1) |= cstr] where [n] is the length of
+      the context and [cstr] the abstracted constraints. *)
+
   val empty : t
+  val is_empty : t -> bool
 
+  (** Don't use. *)
   val instance : t -> Instance.t
-                        
+
```

Comment:
I'm currently getting rid of the remaining uses. Most of them can be replaced by abstract contexts, including in canonical projections. The only problematic uses are printing.